src/simtools/applications/production_scale_events.py

```diff
@@ -119,66 +106,22 @@ def _parse(label, description):
         default=("prod6_LaPalma-{zenith}deg_gamma_cone.N.Am-4LSTs09MSTs_ID0_reduced.fits"),
         help=(
             "Template for the file name. (default: "
```

Comment:
This help message is incomplete - what is "the file name"?

Reply:
added